### PR TITLE
add dropdown to select version

### PR DESCRIPTION
**Pull request by chenxsan**
Say we're on page https://webpack.js.org/concepts/entry-points/, and we want to check the respective v4 version, there's no easy way to make it at the moment.

We should click the v4 link first in the sidebar:

Then click the `Documentation` menu at the top, then `concepts`, then `Entry Points` in the sidebar:

image

There're four steps to make it.

This pull request introduces a dropdown to make it in only one step:

image

## Notice

I'm introducing tailwindcss for styling in this pull request, here's why:

1. Contributors could reuse those css classes defined in `tailwind.config.js` which is derived from webpack [branding guide](https://webpack.js.org/branding/). Defining css classes in scss files is difficult as naming is hard, and we might easily bring in some new colors against the branding guide without notice.
2. It adds about 2kB (see below)  to the `index.xxx.css` which is all right in my opinion because extending `Sidebar.scss` would add size as well.
    > .static{position:static}.relative{position:relative}.z-10{z-index:10}.box-border{box-sizing:border-box}.block{display:block}.inline{display:inline}.flex{display:flex}.inline-flex{display:inline-flex}.w-full{width:100%}.transform{--tw-translate-x:0;--tw-translate-y:0;--tw-rotate:0;--tw-skew-x:0;--tw-skew-y:0;--tw-scale-x:1;--tw-scale-y:1;transform:translateX(var(--tw-translate-x)) translateY(var(--tw-translate-y)) rotate(var(--tw-rotate)) skewX(var(--tw-skew-x)) skewY(var(--tw-skew-y)) scaleX(var(--tw-scale-x)) scaleY(var(--tw-scale-y))}.resize{resize:both}.appearance-none{-webkit-appearance:none;-moz-appearance:none;appearance:none}.flex-col{flex-direction:column}.rounded-none{border-radius:0}.border{border-width:1px}.border-solid{border-style:solid}.border-gray-200{--tw-border-opacity:1;border-color:rgba(222,222,222,var(--tw-border-opacity))}.bg-white{--tw-bg-opacity:1;background-color:rgba(255,255,255,var(--tw-bg-opacity))}.px-\[5px\]{padding-left:5px;padding-right:5px}.py-\[5px\]{padding-bottom:5px;padding-top:5px}.text-14{font-size:14px}.text-gray-600{--tw-text-opacity:1;color:rgba(83,83,83,var(--tw-text-opacity))}.shadow{--tw-shadow:0 1px 3px 0 rgba(0,0,0,0.1),0 1px 2px 0 rgba(0,0,0,0.06);box-shadow:var(--tw-ring-offset-shadow,0 0 transparent),var(--tw-ring-shadow,0 0 transparent),var(--tw-shadow)}.filter{--tw-blur:var(--tw-empty,/*!*/ /*!*/);--tw-brightness:var(--tw-empty,/*!*/ /*!*/);--tw-contrast:var(--tw-empty,/*!*/ /*!*/);--tw-grayscale:var(--tw-empty,/*!*/ /*!*/);--tw-hue-rotate:var(--tw-empty,/*!*/ /*!*/);--tw-invert:var(--tw-empty,/*!*/ /*!*/);--tw-saturate:var(--tw-empty,/*!*/ /*!*/);--tw-sepia:var(--tw-empty,/*!*/ /*!*/);--tw-drop-shadow:var(--tw-empty,/*!*/ /*!*/);filter:var(--tw-blur) var(--tw-brightness) var(--tw-contrast) var(--tw-grayscale) var(--tw-hue-rotate) var(--tw-invert) var(--tw-saturate) var(--tw-sepia) var(--tw-drop-shadow)}.dark .dark\:bg-gray-100{--tw-bg-opacity:1;background-color:rgba(242,242,242,var(--tw-bg-opacity))}

### CODE REVIEW

**Comment by snitin315** [comment]
Good move 👍

File: src/components/Sidebar/Sidebar.jsx [hunk 1]
```diff
@@ -1,44 +1,53 @@
...
-          <a href="https://github.com/webpack/webpack/releases">
-            <Shield
-              content="github/package-json/v/webpack/webpack"
-              label="webpack"
-            />
-          </a>
-        </div>
+        <select
+          name="version-selector"
+          style={{
```
Comment (montogeek) [inline src/components/Sidebar/Sidebar.jsx:36]: Please use a class name.

**Comment by montogeek** [comment]
It will be good to provide the same for the v4 docs. Otherwise we will be redirecting users from the latest docs version.

**Comment by chenxsan** [comment]
Yeah, will backport the feature once it lands in v5.

**Pull request merged**